packages/validator/src/services/utils.ts

```diff
@@ -59,3 +80,38 @@ export function groupSyncDutiesBySubCommitteeIndex(
 
   return dutiesBySubCommitteeIndex;
 }
+
+/**
+ * Return signature in bytes. Assumption that the pubkey has it's corresponding secret key in the keystore of the remote signer.
+ */
+export async function requestSignature(
+  pubkey: string | BLSPubkey,
+  signingRoot: string | Uint8Array,
+  endpoint: string
+): Promise<Uint8Array> {
+  await init("blst-native");
```

Comment:
We can't have this `init` here. The `init` has to happen at some higher level (like the CLI)

Reply:
The only reason i was using init() here was that I needed to convert signature from a hex string to its byte representation but i realized I could do the conversion directly. I'm now using fromHexString() function from @chainsafe/ssz to do the conversion.
https://github.com/ChainSafe/lodestar/pull/3502/commits/2acafd8f629dac746511450846d667d7df0b07f2